### PR TITLE
Addresses #199 - removing non-functioning layer switcher for now

### PR DESCRIPTION
**Pull request by NatHillardUSDS**
Previously part of demo branch, now breaking out as part of its own fix for clarity. There's definitely more to do for #199, but needed this for map display for now

### CODE REVIEW

File: client/src/pages/cejst.tsx [hunk 1]
```diff
@@ -14,7 +13,7 @@ interface IMapPageProps {
 }
 
 const CEJSTPage = ({location}: IMapPageProps) => {
-  const [features, setFeatures] = useState<Feature<Geometry>[]>([]);
+  const [features] = useState<Feature<Geometry>[]>([]);
```
Comment (TomNUSDS) [inline client/src/pages/cejst.tsx:16]: Humm.. `useState` without a setter? I think that just makes it a `const`?
Is this just a temporary thing so you can easily remove the `<MapControls>`

Comment (NatHillardUSDS) [reply client/src/pages/cejst.tsx:16]: This is temporary for now, yes - we might need it later

Comment (TomNUSDS) [reply client/src/pages/cejst.tsx:16]: OK. Maybe a comment explaining that and then the PR LGTM.

**Review by TomNUSDS — DISMISSED** [review]
LGTM

File: client/src/pages/cejst.tsx (hunk 1 above)
Comment (NatHillardUSDS) [reply client/src/pages/cejst.tsx:19]: Good call - added comment just now

**Review by TomNUSDS — APPROVED** [review]
Reapproved

**Pull request merged**